duckduckgo/macos-browser
# Updates BSK to 68.0.0

Task/Issue URL: https://app.asana.com/0/1203512625915051/1205074419464907/f

BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/419
iOS PR: https://github.com/duckduckgo/iOS/pull/1837

## Description:

Network Protection module cleanup:
- We're not using the `CFMessagePort` IPC mechanism so we can delete the code for that.
- We're not using the distributed notifications code in iOS, so we can make that code platform-specific.

## Testing:

### Cleanup before testing

For these tests it's important that you're running the latest menu agent.

1. Run `launchctl list | grep duck`
2. Run `launchctl remove <agent_app_id>`

### Test

1. Run the app
3. Connect NetP
4. Make sure the connection works
5. Make sure the main and agent apps show the right server selection and connection status.
6. Try changing server and see that the server information is properly updated.

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
